converters/google/resources/project.go

```diff
@@ -34,7 +34,7 @@ func GetProjectCaiObject(d TerraformResourceData, config *Config) ([]Asset, erro
 	}
 	name, err := assetName(d, config, linkTmpl)
 	if err != nil {
-		return []Asset{}, err
+		name = fmt.Sprintf("//cloudresourcemanager.googleapis.com/projects/placeholder-%s", randString(8))
```

Comment:
In the long run, this might need to be fixed at a higher level - the project and billing info both get created from the same resource data, which means that they should technically have the same project ID. But in practice, it probably doesn't matter since they're separate asset types and (presumably) haven't been created yet.

For now - fixing this in assetName should be fine.

Reply:
I handled this in transport.go, after trying all kinds of getting project failed, it returns a `placeholder-project` string, while terraform data's project id is still an empty string. In ancestry manager, if terraform data project id is empty, then trigger the org/unknown handling.

Reply:
That probably would work - but unfortunately, we can't make that change in transport.go because it's [shared with the Terraform provider](https://github.com/hashicorp/terraform-provider-google/blob/main/google/transport.go). Could it happen in assetName instead?

Reply:
Added a workaround there to temporarily set the config's project, and unset it later, else need to copy the build template function over?